libraries/ESP8266HTTPClient/examples/StreamHttpsClient/StreamHttpsClient.ino

```diff
@@ -0,0 +1,116 @@
+/**
+   StreamHTTPClient.ino
+
+    Created on: 24.05.2015
+
+*/
+
+#include <Arduino.h>
+
+#include <ESP8266WiFi.h>
+#include <ESP8266WiFiMulti.h>
+
+#include <ESP8266HTTPClient.h>
+
+#define USE_SERIAL Serial
+
+ESP8266WiFiMulti WiFiMulti;
+
+void setup() {
+
+  USE_SERIAL.begin(115200);
```

Comment:
I am not sure that Serial1 is such a common use case to justify USE_SERIAL. I think using `Serial` makes for more obvious examples which are easier to copy-paste and combine. (This applied to many existing examples, just wanted to give an opinion.)

Reply:
USE_SERIAL was already present in the examples of the current 1.1 version of the library. Replacing USE_SERIAL with Serial in all examples